RPIPs/RPIP-53.md

```diff
@@ -0,0 +1,110 @@
+
+This **does not** require a contract change (ironically, it's more in-line with how the contracts actually behave in Houston) and can be implemented at any time after Houston has launched.
+
+
+## Motivation
+
+The Redstone update introduced the Merkle-tree-based rewards system, which allowed users to claim RPL they've earned for both node operating duties and Oracle DAO duties, as well as their share of Smoothing Pool rewards. Since its inception, this system has gone through several revisions to account for fixes and Ethereum upgrades but has fundamentally remained the same: the tree lists rewards earned for individual node operators, and node operators can claim those rewards at any time.
+
+On that note, Houston (or more specifically RPIP-31) has an unfortunate consequence: if you have an RPL withdrawal address set, [you can no longer claim rewards intervals](https://github.com/rocket-pool/rocketpool/blob/a747457fc610aa889af0a13cff5d3d00955525a5/contracts/contract/rewards/RocketMerkleDistributorMainnet.sol#L85). This includes both RPL rewards and Smoothing Pool ETH rewards. Claiming them *must* be done from the RPL withdrawal address. Whether intended or not, the RPL withdrawal address isn't really an RPL withdrawal address in Houston; it's a "claimer address" that can claim rewards intervals from the Merkle distributor contract.
+
+This is problematic for a few reasons. As an example: the node is still going to be listed in the rewards file, and thus the rewards for the corresponding interval are still going to show up as a "claimable interval" in the Smart Node despite the user not being able to claim from it, and not being able to actually *receive* any of the rewards from that interval. It will only disappear from the list once the RPL withdrawal address has claimed from it. This can obviously be worked around in the Smart Node but it's exactly that: a workaround. The file itself is still going to have the node operator listed, so anyone (such as a third party) that wants to use it to view or claim rewards has to know about and implement this workaround as well.
+
+In this RPIP, I'm proposing a small change to the rewards file that ameliorates the problems and provides some helpful benefits for people that would actually take advantage of the feature in the first place.
+
+
+## Important Clarification
+
+Note that the RPL withdrawal address, as specified in RPIP-31, is **not** the same as the "primary withdrawal address" (previously known as just the "withdrawal address" in Atlas and below). If a node operator has a "regular" withdrawal address set, they will *not be affected by this change*. They can still claim rewards normally. The only nodes affected by this change are ones that explicitly set an **RPL withdrawal address**, which is a different setting than the "primary withdrawal address", after Houston has been deployed.
+
+
+## Specification
```

Comment:
I'm not happy for a Protocol RPIP to have its Specification given as an amendment only. I think it's good to include the amendment in the RPIP, but for the Specification itself to be a full copy of the specification (probably as an asset attached, as with the previous RPIPs specifying rewards).

Reply:
As stated in the Specification section:

> This change itself is orthogonal to the decision of whether or not v9 is ratified, as v9 is a non-functional format change from JSON to SSZ, but the specification itself does depend on whether or not v9 is ratified so it can build upon the proper baseline. I'm calling this "v10" because I anticipate v9 will go through and come first. I'm happy to formalize the specification once the decision has been made. In the mean time, I'll propose the changes here at a high-level.
> 
> Alternatively I can adjust both the v8 and v9 specs accordingly, just so we have them ready to go. I defer to the community to decide whether or not that's prudent. Anyway, high-level changes for now:

I was just waiting to see whether or not v9 would be formalized first, since the spec changes based on whether or not it is and it's still in flight. Sounds like you're lobbying for the "do both specs and we'll pick whichever one is relevant at the time" camp, which is fine; it'll just take a little longer to write and I wanted to get this out before that.